OpenRA.Game/Server/Server.cs

```diff
@@ -743,20 +743,39 @@ public void DispatchOrdersToClients(Connection conn, int frame, byte[] data)
 				if (c != conn && c.Validated)
 					DispatchFrameToClient(c, from, frameData);
 
+			RecordOrder(frame, data, @from);
```

Comment:
Rider is saying that the verbatim prefixes here and below are redundant, so can we please drop these?